bionic/task_state.py

```diff
@@ -252,33 +275,18 @@ def sync_after_subprocess_completion(self):
 
         assert self.should_persist
 
-        # First, let's flush the stored entries in cache accessors. This is to prevent cases
-        # like current process contains an outdated stored entries.
-        #
-        # For example: In assisted mode, main process checks for versioning error which can
-        # populate the stored entries as entries without an artifact. Now subprocess can
-        # update the stored entries after computing them but it won't be communicated back.
-        # This allows the main process to fetch fresh entries which might have been updated.
+        # First, let's flush the stored entries in cache accessors. Since we just
```

Comment:
I think we can replace this method with `refresh_all_cache_state`.

Reply:
Discussed this offline; we'll leave it for now.